.github/workflows/ci.yml

```diff
@@ -60,8 +60,8 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.6.x", "3.7.x"]
-        engine: ["ray"]
-        part: [3]
+        engine: ["python", "ray", "dask"]
+        part: ["Reduction_A", "Reduction_B", "Binary", "MapMetadata", "UDF", "Default", "Window", "Indexing", "Iter", "JoinSort", 3]
```

Comment:
1) What does the part with "3" name mean?

2) Why testing of "test_dataframe.py" happens via classes? Have you thought about splitting the file into several other files?

Reply:
1. Part 3 means everything but dataframe.

2. Running via several files may make sense in the future. I want to avoid making it confusing for adding new tests, the question "Where do the new tests go?" or "Where are the tests for `DataFrame.pivot`"? will be hard to answer individually every time. I'm happy to hear your thoughts about this, but this was my main thinking.

Reply:
1. Can we name part "3" something like "everything_except_dataframes"?
2. One way or another, contributors must choose the class in which to add the test. If we create a file for each class and create a folder for all files, nothing will change cardinally. They will continue to choose a place not among the classes, but among the files in the folder. These are the first thoughts :)

Reply:
I see, your reasoning makes sense. I would prefer any changes fall into a new PR since the aim of this is to re-enable GitHub Actions. If you can open a new issue for discussion that would be great. We can try to discuss there what the right breakdown of the files should be.

Reply:
Ok, #1527 was be created for following discussion.